### PR TITLE
docs: add code block language for fix website

### PR DESCRIPTION
**Pull request by datamweb**
Hello,
PR are created to improve the display of the code block on the site and to facilitate the copying of commands by the user.
see : https://github.com/codeigniter4/shield/pull/508#issuecomment-1305485529
see : https://codeigniterchat.slack.com/archives/C47DHAS7M/p1668107201982039?thread_ts=1668040926.000199&cid=C47DHAS7M

### CODE REVIEW

**Review by paulbalandan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -54,8 +54,9 @@ Usage of Shield requires the following:
 ### Installation
 
 Installation is done through Composer.
-
-    > composer require codeigniter4/shield
+```console
```
Comment (MGatner) [inline README.md:57]: `console` is for shell output, `shell` is for commands: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

File: docs/install.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ These instructions assume that you have already [installed the CodeIgniter 4 app
 Installation is done through [Composer](https://getcomposer.org). The example assumes you have it installed globally.
 If you have it installed as a phar, or otherwise you will need to adjust the way you call composer itself.
 
-```
-> composer require codeigniter4/shield
+```console
```
Comment (MGatner) [inline docs/install.md:28]: Shell

File: docs/install.md [hunk 3]
```diff
@@ -35,29 +35,29 @@ If you have it installed as a phar, or otherwise you will need to adjust the way
...
     ```json
-        "minimum-stability": "dev",
-        "prefer-stable": true,
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     ```
 
 2. Or specify an explicit version:
 
     ```console
```
Comment (MGatner) [inline docs/install.md:52]: Shell

File: docs/install.md [hunk 4]
```diff
@@ -35,29 +35,29 @@ If you have it installed as a phar, or otherwise you will need to adjust the way
...
-    > composer require codeigniter4/shield:dev-develop
+    composer require codeigniter4/shield:dev-develop
     ```
 
     The above specifies `develop` branch.
     See https://getcomposer.org/doc/articles/versions.md#branches
 
     ```console
```
Comment (MGatner) [inline docs/install.md:59]: Shell

File: docs/install.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ If you get the following error:
 1. Run the following command. This command handles steps 1-5 of *Manual Setup* and runs the migrations.
 
     ```console
```
Comment (MGatner) [inline docs/install.md:72]: Shell

File: docs/install.md [hunk 6]
```diff
@@ -120,7 +120,7 @@ your project.
 5. **Migration** Run the migrations.
 
     ```console
```
Comment (MGatner) [inline docs/install.md:122]: Shell

**Review by MGatner — CHANGES_REQUESTED** [review]
Thanks for adding these! I'm not sure if MkDocs uses the same aliases as GitHub but they differentiate she'll commands from terminal output: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Maybe test one or two before going through the trouble of changing them all.

File: README.md (hunk 1 above)
Comment (kenjis) [reply README.md:57]: In my understanding, `console` is shell session. 
All characters including command prompts and commands and outputs in the terminal.

`shell` is a shell script that is program code written by shell.

Comment (MGatner) [reply README.md:57]: That's right. Console is actually an alias for ShellSession:
```yml
ShellSession:
  type: programming
  extensions:
  - ".sh-session"
  aliases:
  - bash session
  - console
```

@kenjis Are you saying I have misinterpreted how this should be used?

Comment (kenjis) [reply README.md:57]: I think if we copy and paste the terminal output, we use `console`.
If it is shell script, we use `shell`.

So I am Okay with this PR.

**Review by MGatner — APPROVED** [review]

**Comment by kenjis** [comment]
Before:
Screenshot 2022-11-15 11 42 26

After:
Screenshot 2022-11-15 11 42 31

**Review by kenjis — APPROVED** [review]

**Pull request merged**